wormhole-connect/src/hooks/useSortedRoutesWithQuotes.ts

```diff
@@ -134,3 +136,24 @@ export const useSortedRoutesWithQuotes = (): HookReturn => {
     ],
   );
 };
+
+const bucketByEta = (
+  routesWithQuotes: RouteWithQuote[],
+): RouteWithQuote[][] => {
+  const thresholds = [60 * 1000, Infinity];
+  const buckets: RouteWithQuote[][] = [];
+
+  for (const routeAndQuote of routesWithQuotes) {
+    const { quote } = routeAndQuote;
+    for (let i = 0; i < thresholds.length; i++) {
+      const threshold = thresholds[i];
+      if (quote.eta && quote.eta <= threshold) {
+        if (buckets[i] === undefined) buckets[i] = [];
```

Comment:
If we set `buckets[1]`, but never `buckets[0]`, then I think this call will throw when accessing `buckets[0]` which is `undefined`: https://github.com/wormhole-foundation/wormhole-connect/pull/2934/files#diff-0254f3d56b61ab2784afef85542f31c5480c00774bc2d46a93358cdd8077af6cR74

Reply:
oh good catch

Reply:
fixed in eef1044